src/pages/inventory/inventory.html

```diff
@@ -26,23 +26,23 @@
   </ion-segment>
 
   <ion-item>
-    <ion-label floating>Brand</ion-label>
+    <ion-label color="primary">Brand</ion-label>
```

Comment:
This is a little weird, but inline labels (default if none is specified) look better with drop downs. With fixed labels, the drop down value is placed in the middle of the input, with inline labels, the input is all the way to the right.